Gemfile

```diff
@@ -28,7 +30,6 @@ gem "unicorn"
 group :development do
   gem "spring"
   gem "spring-commands-rspec"
-  gem "web-console"
```

Comment:
You should try to add this back in. If it gives you trouble, don't worry about it, but if it works, I think it's supposed to be pretty cool.